.github/actions/isStagingDeployLocked/isStagingDeployLocked.js

```diff
@@ -4,13 +4,15 @@ const GithubUtils = require('../../libs/GithubUtils');
 
 const run = function () {
     return GithubUtils.getStagingDeployCash()
-        .then(({labels}) => {
+        .then(({labels, number}) => {
             console.log(`Found StagingDeployCash with labels: ${_.pluck(labels, 'name')}`);
             core.setOutput('IS_LOCKED', _.contains(_.pluck(labels, 'name'), '🔐 LockCashDeploys 🔐'));
+            core.setOutput('NUMBER', number);
```

Comment:
You'll need to add this new output to the [action metadata file](https://github.com/Expensify/Expensify.cash/blob/9f52ebf6423c9396cad8f7a1ff066cde41b3b18f/.github/actions/isStagingDeployLocked/action.yml).